mthamayor/auto-mart
# feature(frontend):improve dashboard view

#### What does this PR do?
Improve the dashboard view

#### Description of Task to be completed?
- The following pages should consume endpoints
1. https://mthamayor.github.io/auto-mart/dashboard.html

- Users should be able to view all their adverts and created purchase orders with the following endpoints
1. GET /order/user/my-orders
2. GET /car/user/my-cars
 
View [API documentation](https://mthamayor-auto-mart.herokuapp.com/api/v1/docs/#/) for endpoints specification

#### How should this be manually tested?
- Visit the dashboard page https://mthamayor.github.io/auto-mart/dashboard.html
- You should see a list of your purchase orders and adverts if you have any

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167085273

#### Screenshots (if appropriate)
None

#### Questions:
None
